app/src/features/settings/components/BillingTeam/components/BillingDetails/modals/common/CancelPlanModal/index.tsx

```diff
@@ -35,63 +43,59 @@ export const CancelPlanModal: React.FC<Props> = ({ isOpen, closeModal, subscript
         toast.warn("Please enter a valid reason for cancellation!");
         return;
       }
+    }
 
-      setIsLoading(true);
+    setIsLoading(true);
 
-      const cancelIndividualSubscription = httpsCallable(getFunctions(), "subscription-cancelIndividualSubscription");
+    const cancelSubscription = httpsCallable<
+      {
+        reason: string;
+        currentPlan: string;
+        billingId: string;
+      },
+      {
+        success: boolean;
+        message: string;
+      }
+    >(getFunctions(), "subscription-cancelSubscription");
```

Comment:
Lets move this inside `backend/billing` similar to [this](https://github.com/requestly/requestly/blob/master/app/src/backend/billing/requestTeamAccess.ts)